README.md

```diff
@@ -161,7 +161,10 @@ This repository is a [monorepo](https://trunkbaseddevelopment.com/monorepos/) ma
 
 ### Contributing to Gatsby v2
 
-We are currently only accepting bug fixes for Gatsby v2. No new features will be accepted.
+The maintenance window for Gatsby v2 closes on April 1, 2021.
+
+- During this maintenance window, we will continue to address reported bugs in Gatsby v2.
+- After this maintenance window, we will only accept critical security updates for Gatsby v2, and feature requests will be considered for Gatsby V3
```

Comment:
It's been a _hot_ minute since we've been open to feature requests—and our process(es) have changed significantly since then. Should we point/link over to Discussions?

Reply:
@janson Thanks for the feedback

If you think it's better to point users to discussions specifically, feel free to propose a specific change. I wasn't calling out a specific method of submitting feature requests. Today, these come by way of discussions / pull requests, and out of band.

The key takeaway that's important for our users to understand is that we'll not be adding new features to V2 after April 1, 2021.

Given that, what specific change would you suggest?